Issue 1199 - Misleading message in access log for idle timeout

jchapma opened the pull request:
Description:
Update timeout error code in daemon.
Add extra detail to idle and IO timeout error messaging.
Typo in logconv.pl

Relates: #1199

Reviewed by:

jchapma committed:
Issue 1199 - Misleading message in access log for idle timeout

Description:    Update timeout error code in daemon.
                Add extra detail to idle and IO timeout error messaging.
                Typo in logconv.pl

Relates:        #1199

Reviewed by:    mreynolds389, droideck, Firstyear (Thanks folks)

jchapma committed:
Issue 1199 - Misleading message in access log for idle timeout

Description:    Update timeout error code in daemon.
                Add extra detail to idle and IO timeout error messaging.
                Typo in logconv.pl

Relates:        #1199

Reviewed by:    mreynolds389, droideck, Firstyear (Thanks folks)

droideck reviewed (approved):
Joining Mark's comment and the rest - looks good to me!

jchapma committed:
Indentation

jchapma committed:
Issue 1199 - Misleading message in access log for idle timeout

Description:    Update timeout error code in daemon.
                Add extra detail to idle and IO timeout error messaging.
                Typo in logconv.pl

Relates:        #1199

Reviewed by:    mreynolds389, droideck, Firstyear (Thanks folks)
